index.html

```diff
@@ -14,20 +14,170 @@
   <header class="header">
     <nav class="navbar">
       <a class="logo navbar-item position"><img src="images/image_geometry_1.svg" alt="A geometry image"/></a>
-      <i class="fas fa-bars navbar-item icon position"></i>
+      <i class="fas fa-bars navbar-item icon position primary-color"></i>
     </nav>
   </header>
   <section class="summary">
-    <div class="name position left16 align-left no-margin">
+    <div class="name position left16 align-left no-margin primary-color">
       <h1 class="no-margin">Ricky
       Mormor</h1>
     </div>
-    <div class="intro position left16 align-left width288">
+    <div class="intro position left16 align-left width288 poppins">
       <p>Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
         If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
       </div>
 
-      <button class="cta position typography width288">Start collaboration</button>
+    <button type="button" class="cta position typography width288 primary-color">Start collaboration</button>
+  </section>
+
+  <section class="projects-section no-margin">
+    <div class="head-image position"></div>
+    <div class="primary-color projects-head position typography left16">
+      <h2>Projects</h2>
+    </div>
+
+    <div class="projects-list no-margin">
+      <div class="project-cards no-margin">
+        <div class="cards">
+          <div class="card1 fix-card"></div>
+          <div class="cards-detail dark-bg">
+            <p class="poppins">name goes here</p>
+            <button type="button">HTML/CSS</button>
+            <button type="button">Ruby on Rails</button>
+            <button type="button">Javascript</button>
```

Comment:
- [ ] [OPTIONAL] I think it could be good to group these technologies together as a list, and they don't actually need to be buttons since we don't expect anything to happen if we click on them, so keeping them inside a `<li>` makes more sense, see the example below 👇 
```
<ul>
   <li>HTML/CSS</li>
   <li>Ruby on Rails</li>
</ul>
```